site/app/templates/submission/regrade/Discussion.twig

```diff
@@ -92,6 +92,10 @@
                                 </button>
                             {% elseif grade_inquiries[component.id].status == 0 and is_grading %}
                                 {# Grader view, Request closed #}
+                                {# Lettting Instructor reply to the inquiry without reopening the grade_inquiry #}
+                                <button disabled type="button" formaction="{{ make_request_post_url }}" class="gi-submit btn btn-primary" style="margin:15px 0 0 5px; float: right" onclick="onGradeInquirySubmitClicked(this)">
```

Comment:
Why is this button disabled?

Perhaps it would make it more clear to graders if this button was labeled "comment".

Reply:
@shwehtom89 button is kept `disabled` or `enabled` based on the fact if there is something in the textbox to submit or not. So by default it is `disabled` and gets enabled once you type in something.
For Label I have updated it with **Add Comment**